app.js

```diff
@@ -1,4 +1,33 @@
 //  in this file create an express application - use the middle-ware built into express
 //  to serve up static files from the public directory (index.html and client.js - you 
 //  can also serve up css files from the public directory this way if you want)
-//  you need to support a '/trucks' endpoint, and a dynamic route for '/trucks:name'
\ No newline at end of file
+//  you need to support a '/trucks' endpoint, and a dynamic route for '/trucks:name'
+'use strict';
+var express = require('express');
+var trucks = require('./trucks');
+var app = express();
+
+app.use(express.static('public'));
+app.get('/', function (request, response) {
+  response.redirect('/trucks');
+});
+
+app.get('/trucks', function (request, response) {
+  response.json(trucks.getTrucks());
+});
+
+app.get('/trucks/:name', function (request, response) {
+  response.json(trucks.getTruck(request.params.name));
```

Comment:
If no truck is returned from getTruck, it would be better to return a 404 status code and inform the reponse that the resource wasn't found.